src/cljs/rems/application.cljs

```diff
@@ -89,7 +89,7 @@
  ::enter-application-page
  (fn [{:keys [db]} [_ id]]
    {:db (-> db
-            (assoc ::application-id id)
+            (assoc ::application-id (parse-int id))
```

Comment:
Is the potential error here handled correctly?

Reply:
I.e. does the error page make sense for this one? I can see someone hand-typing the app-id sometime (or maybe it would be mostly for the external app ids in urls)

Reply:
if I go to `/application/abc`, I get a red box saying

> Application: Failed
> Bad Request

Reply:
Maybe it should be 404 like with the numbers?

Reply:
I'm fine with this one too since it's technically a bad request. Too bad that there is nothing about the app id there.